tools/preflight/cleanup_test.go

```diff
@@ -0,0 +1,71 @@
+package preflight
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/trilioData/tvk-plugins/internal"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var _ = Describe("Cleanup Unit Tests", func() {
```

Comment:
need test-cases for func `CleanupPreflightResources`

Reply:
Will add test-cases for `CleanupPreflightResources` when DNS and volume snapshot resouces checks for env cluster will be added.